streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -43,7 +43,7 @@ import org.apache.spark.storage.StorageLevel
 import org.apache.spark.streaming.StreamingContextState._
 import org.apache.spark.streaming.dstream._
 import org.apache.spark.streaming.receiver.Receiver
-import org.apache.spark.streaming.scheduler.{JobScheduler, StreamingListener}
+import org.apache.spark.streaming.scheduler.{ExecutorAllocationManager, JobScheduler, StreamingListener}
```

Comment:
Is the `ExecutorAllocationManager` import necessary? It doesn't seem to be referenced here.